gui-apps/foot/foot-1.23.0.ebuild

```diff
@@ -15,7 +15,7 @@ SRC_URI="
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~arm64 ~ppc64 ~riscv ~x86"
-IUSE="+grapheme-clustering test utempter verify-sig"
```

Comment:
Man pages must be installed unconditionally: https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0305.